runtime/ftplugin/python.vim

```diff
@@ -118,32 +118,24 @@ if !exists("g:python_recommended_style") || g:python_recommended_style != 0
     setlocal expandtab shiftwidth=4 softtabstop=4 tabstop=8
 endif
 
-" First time: try finding "pydoc".
 if !exists('g:pydoc_executable')
-    if executable('pydoc')
-        let g:pydoc_executable = 1
-    else
-        let g:pydoc_executable = 0
-    endif
+  let g:pydoc_executable = has('win32') ? 'python -m pydoc' : 'pydoc'
 endif
-
-" Windows-specific pydoc setup
-if has('win32') || has('win64')
-    if executable('python')
-        " available as Tools\scripts\pydoc.py
-        let g:pydoc_executable = 1
-    else
-        let g:pydoc_executable = 0
-    endif
-endif
-
-" If "pydoc" was found use it for keywordprg.
-if g:pydoc_executable
-    if has('win32') || has('win64')
-        setlocal keywordprg=python\ -m\ pydoc\ 
-    else
-        setlocal keywordprg=pydoc
-    endif
+if executable(split(g:pydoc_executable)[0])
+  if !has('gui_running')
+    command! -buffer -nargs=1 RunHelp silent exe '!' . g:pydoc_executable . ' "<args>" 2>/dev/null' | redraw!
+  elseif has('terminal')
+    command! -buffer -nargs=1 RunHelp silent exe ':term ' . g:pydoc_executable . ' "<args>"'
+  else
+    command! -buffer -nargs=1 RunHelp echo system(g:pydoc_executable . ' "<args>" 2>/dev/null')
+  endif
+  setlocal keywordprg=:RunHelp
+
+  if !exists('b:undo_ftplugin') || empty(b:undo_ftplugin)
+    let b:undo_ftplugin = 'setlocal keywordprg< iskeyword<'
+  else
+    let b:undo_ftplugin .= '| setlocal keywordprg< iskeyword<'
+  endif
```

Comment:
Just add to `b:undo_ftplugin` at the end of the file?

Reply:
This was already included by the maintainer